src/hotspot/os_cpu/linux_riscv/riscv_hwprobe.cpp

```diff
@@ -172,12 +172,27 @@ void RiscvHwprobe::add_features_from_query_result() {
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZBB)) {
     VM_Version::ext_Zbb.enable_feature();
   }
+  if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZBC)) {
+    VM_Version::ext_Zbc.enable_feature();
+  }
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZBS)) {
     VM_Version::ext_Zbs.enable_feature();
   }
   if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZFH)) {
     VM_Version::ext_Zfh.enable_feature();
   }
+  if (is_set(RISCV_HWPROBE_KEY_IMA_EXT_0, RISCV_HWPROBE_EXT_ZICBOZ)) {
```

Comment:
Could you add Zicbop and Zicbom as well? Are they even supported/defined in the kernel headers?

Reply:
Seems they are not supported in kernel yet, please check https://github.com/torvalds/linux/blob/master/arch/riscv/include/uapi/asm/hwprobe.h.